pyanaconda/modules/network/network.py

```diff
@@ -402,6 +402,17 @@ def collect_requirements(self):
                 reason="Necessary for biosdevname network device naming feature."
             ))
 
+        # encrypted dns
```

Comment:
What about the RHEL 10/9 bootc images ? Do they have/need this as well by default ? CC @cgwalters

Reply:
Like, we can't really add stuff to them during the installation live we can add extra RPMs during an RPM based installation.